filebeat/module/apache2/access/_meta/fields.yml

```diff
@@ -4,30 +4,31 @@
     Contains fields for the Apache2 HTTPD access logs.
   fields:
     - name: remote_ip
-      type: keyword
-      description: >
-        Client IP address.
+      type: alias
+      path: source.ip
+
     - name: user_name
-      type: keyword
-      description: >
-        The user name used when basic authentication is used.
+      type: alias
+      path: user.name
+
     - name: method
-      type: keyword
-      example: GET
-      description: >
-        The request HTTP method.
+      type: alias
+      path: http.request.method
+
     - name: url
-      type: keyword
-      description: >
-        The request HTTP URL.
+      type: alias
```

Comment:
@webmat Some examples on how the alias fields will be defined. I image we could even allow users to disable them if they do not want to see them.